### ISSUE TITLE
training data list

### ISSUE THREAD

**Issue by taalua** [Respaired/Tsukasa-Speech#3]
Hi,
Thank you for this great work.
How did you associate speaker and text in the form of "speaker: text". I saw you used logits from "line-corporation/line-distilbert-base-japanese" . Would you mind sharing what your training data list .txt for multispeaker fine-tuning looks like, i.e., snippet?

Also, how to train the Kotodama-Sampler myself?

Kind regards,

**Comment by Respaired** [comment]
Hey.

it's a mapping of the style vectors to the text, by adding a regression head on top of a bert model.
the data looks like:

```こんいちは、せかい！, [0.24125,0.012551,.025125125 .....] (shape = 256)```
while Kotodama sampler is a big advantage of Tsukasa, I consider it very experimental as it has some quirks that I need to address later. you don't necessarily need it to train your model.

**Comment by taalua** [comment]
Hello,

Thank you for the prompt resonse.
So, no need for speaker name or speaker index in the training data list .txt ?

Thanks.

**Comment by Respaired** [comment]
sorry, i was busy for a week didn't see this.

yes, you can map the vectors to any format you'd like.
